README.md

```diff
@@ -121,5 +121,169 @@ Run the next commands
     - Replace the `workflow_id` with the id returned by the previous response.
     - A response with the workflow status will be returned.
  
+## File upload process in Cromwell Pipeline
+### Step 1: WDL file validation
+For validation WDL file send POST request <br>
+to address `<cromwell_pipeline_url>/files/validation` <br>
+with header `Authorization <auth_token>` <br>
+and request body:
+```
+{
+    "content": "<file_content>"
+}
+```
+* Instead of `<cromwell_pipeline_url>` insert the application URL
+* Instead of `<auth_token>` insert the authorization token
+* Instead of `<file_content>` insert the content from [cromwell-sample/hello.wdl](https://github.com/EpamLifeSciencesTeam/cmwl_pipeline/blob/a2a82a63ed8c66dc881892c8645e40ee58a36c1a/cromwell-sample/hello.wdl) file
+
+If validation succeed, response OK with HTTP status 200 will be returned.
+### Step 2: Create project (if doesn't exist)
+To create project send POST request <br>
+to address `<cromwell_pipeline_url>/projects` <br>
+with header `Authorization <auth_token>` <br>
+and request body:
+```
+{
+    "name": "<project_name>"
+}
+```
+* Instead of `<cromwell_pipeline_url>` insert the application URL
+* Instead of `<auth_token>` insert the authorization token
+* Instead of `<project_name>` insert the name of new project
+
+If creation succeed, response OK with HTTP status 200 will be returned.
+
+### Get project by name (if exists)
+To get project by name send GET request <br>
+to address `<cromwell_pipeline_url>/projects?name=<project_name>` <br>
+with header `Authorization <auth_token>` <br>
+* Instead of `<cromwell_pipeline_url>` insert the application URL
+* Instead of `<auth_token>` insert the authorization token
+
+If project exists, response OK with HTTP status 200, and the following response body will be returned:
+```
+{
+    "projectId": "<project_id>",
+    "ownerId": "<owner_id>",
+    "name": "<project_name>",
+    "active": true,
+    "repositoryId": <repository_id>,
+    "visibility": "<visibility>"
+}
+```
+### Step 3: Upload file to project
+To upload file into existing project send POST request <br>
+to address `<cromwell_pipeline_url>/files` <br>
+with header `Authorization <auth_token>` <br>
+and request body:
+```
+{
+    "project": {
+        "projectId": "<project_id>",
+        "ownerId": "<owner_id>",
+        "name": "<project_name>",
+        "active": true,
+        "visibility": "<visibility>",
+        "repositoryId": <repository_id>
+    },
```

Comment:
*does not require changes*
Don't forget to update it once another your MR is merged